packages/mdc-theme/_color_palette.scss

```diff
@@ -0,0 +1,290 @@
+$material-color-green-700: #388e3c;
+$material-color-green-800: #2e7d32;
+$material-color-green-900: #1b5e20;
+$material-color-green-a100: #b9f6ca;
+$material-color-green-a200: #69f0ae;
+$material-color-green-a400: #00e676;
+$material-color-green-a700: #00c853;
+
+$material-color-light-green-50: #f1f8e9;
+$material-color-light-green-100: #dcedc8;
+$material-color-light-green-200: #c5e1a5;
+$material-color-light-green-300: #aed581;
+$material-color-light-green-400: #9ccc65;
+$material-color-light-green-500: #8bc34a;
+$material-color-light-green-600: #7cb342;
+$material-color-light-green-700: #689f38;
+$material-color-light-green-800: #558b2f;
+$material-color-light-green-900: #33691e;
+$material-color-light-green-a100: #ccff90;
+$material-color-light-green-a200: #b2ff59;
+$material-color-light-green-a400: #76ff03;
+$material-color-light-green-a700: #64dd17;
+
+$material-color-lime-50: #f9fbe7;
+$material-color-lime-100: #f0f4c3;
+$material-color-lime-200: #e6ee9c;
+$material-color-lime-300: #dce775;
+$material-color-lime-400: #d4e157;
+$material-color-lime-500: #cddc39;
+$material-color-lime-600: #c0ca33;
+$material-color-lime-700: #afb42b;
+$material-color-lime-800: #9e9d24;
+$material-color-lime-900: #827717;
+$material-color-lime-a100: #f4ff81;
+$material-color-lime-a200: #eeff41;
+$material-color-lime-a400: #c6ff00;
+$material-color-lime-a700: #aeea00;
+
+$material-color-yellow-50: #fffde7;
+$material-color-yellow-100: #fff9c4;
+$material-color-yellow-200: #fff59d;
+$material-color-yellow-300: #fff176;
+$material-color-yellow-400: #ffee58;
+$material-color-yellow-500: #ffeb3b;
+$material-color-yellow-600: #fdd835;
+$material-color-yellow-700: #fbc02d;
+$material-color-yellow-800: #f9a825;
+$material-color-yellow-900: #f57f17;
+$material-color-yellow-a100: #ffff8d;
+$material-color-yellow-a200: #ffff00;
+$material-color-yellow-a400: #ffea00;
+$material-color-yellow-a700: #ffd600;
+
+$material-color-amber-50: #fff8e1;
+$material-color-amber-100: #ffecb3;
+$material-color-amber-200: #ffe082;
+$material-color-amber-300: #ffd54f;
+$material-color-amber-400: #ffca28;
+$material-color-amber-500: #ffc107;
+$material-color-amber-600: #ffb300;
+$material-color-amber-700: #ffa000;
+$material-color-amber-800: #ff8f00;
+$material-color-amber-900: #ff6f00;
+$material-color-amber-a100: #ffe57f;
+$material-color-amber-a200: #ffd740;
+$material-color-amber-a400: #ffc400;
+$material-color-amber-a700: #ffab00;
+
+$material-color-orange-50: #fff3e0;
+$material-color-orange-100: #ffe0b2;
+$material-color-orange-200: #ffcc80;
+$material-color-orange-300: #ffb74d;
+$material-color-orange-400: #ffa726;
+$material-color-orange-500: #ff9800;
+$material-color-orange-600: #fb8c00;
+$material-color-orange-700: #f57c00;
+$material-color-orange-800: #ef6c00;
+$material-color-orange-900: #e65100;
+$material-color-orange-a100: #ffd180;
+$material-color-orange-a200: #ffab40;
+$material-color-orange-a400: #ff9100;
+$material-color-orange-a700: #ff6500;
```

Comment:
Is this correct? The color swatches on material.io/guidelines say this is `#ff6d00`.